week4/MealsharingNET/IMealRepository.cs

```diff
@@ -0,0 +1,11 @@
+using MealsharingNET.Models;
+
+namespace MealsharingNET;
+
+public interface IMealRepository
+{
+    IEnumerable<Meal> ListMeals();
+    void Add(Meal m);
+
+    IEnumerable<Meal> GetMeal(int id);
```

Comment:
Similar here as with the reservations, you should have the return type Meal instead of IEnumerable Meal